apis/nucleus/src/components/listbox/components/ListBoxSearch.jsx

```diff
@@ -72,6 +72,7 @@ export default function ListBoxSearch({
   const { translator } = useContext(InstanceContext);
   const [value, setValue] = useState('');
   const [wildcardOn, setWildcardOn] = useState(false);
+  const selectIsDisabled = !!selectionState.selectDisabled?.();
```

Comment:
recommend moving this into each function using it.
to remove the risk of using an old value